examples/tracetest-datadog/tracetest/tracetest.config.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# postgres:
+#   host: tt-postgres
+#   user: postgres
+#   password: postgres
+#   port: 5432
+#   dbname: postgres
+#   params: sslmode=disable
+postgresConnString: "host=tt-postgres user=postgres password=postgres port=5432 sslmode=disable"
```

Comment:
If we don't release the Viper version, only using `postgresConnString` will work for now.